modules/svg/data.js

```diff
@@ -525,7 +526,7 @@ export function svgData(projection, context, dispatch) {
         }, []);
 
         if (!geoPolygonIntersectsPolygon(viewport, coords, true)) {
-            var extent = geoExtent(d3_geoBounds({ type: 'LineString', coordinates: coords }));
+            var extent = new Extent(d3_geoBounds({ type: 'LineString', coordinates: coords }));
```

Comment:
This might be one of the gotchas - [`d3.geoBounds`](https://github.com/d3/d3-geo#geoBounds) returns a 2d array `[[left, bottom], [right, top]]`, and might not map perfectly to the `new Extent` constructor arguments like `min: Vec2, max: Vec2`.  (TypeScript would nag, once we are using it more!)

You might have to do this?  (I could also be totally wrong) 
```suggestion
            var bounds = d3_geoBounds({ type: 'LineString', coordinates: coords });
            var extent = new Extent(bounds[0], bounds[1]);
```